apps/etterlatte-saksbehandling-ui/client/src/components/behandling/brev/Vedtaksbrev.tsx

```diff
@@ -92,7 +92,7 @@ export const Vedtaksbrev = (props: { behandling: IDetaljertBehandling }) => {
       hentBrev(behandlingId, (brev, statusCode) => {
         if (statusCode === 200) {
           setVedtaksbrev(brev)
-        } else if (statusCode === 204) {
+        } else if (statusCode === 204 && redigerbar) {
```

Comment:
Med denne sjekken kan man vel havne i en situasjon der ingenting laster, og vi ikke har et brev. Burde vi hatt en egen visning for dette? F.eks. ved opphør pga. alder / regulering er det ikke noe brev, og hvis vi har avbrutt behandlingen uten noe brev er det heller ikke noe her. Jeg vet ikke om vi har noe for dette allerede?

Reply:
Så på koden før siste push 😅